Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -396,15 +422,19 @@ class FakespotViewController:
 
         case .qualityDeterminationCard:
             let reviewQualityCardView: FakespotReviewQualityCardView = .build()
+            viewModel.reviewQualityCardViewModel.expandState = (fakespotState?.isReviewQualityExpanded ?? false)  ? .expanded : .collapsed
```

Comment:
Instead of a bool can isReviewQualityExpanded be replaced to use the ExpandButtonState enum directly so you don't have to convert it this way?

Reply:
We definitely could but that means we're coupling UI model with business logic model. Is that okay? I was also thinking we could have a different expended state for redux logic but I think it's a bit overkill for this case.